src/host/exe/exemain.cpp

```diff
@@ -16,6 +18,37 @@ TRACELOGGING_DEFINE_PROVIDER(
     (0x770aa552, 0x671a, 0x5e97, 0x57, 0x9b, 0x15, 0x17, 0x09, 0xec, 0x0d, 0xbd),
     TraceLoggingOptionMicrosoftTelemetry());
 
+// Define a specialization of WRL::Module so we can specify a REGCLS_SINGLEUSE type server.
+// We would like to use all the conveniences afforded to us by WRL::Module<T>, but it only
+// creates REGCLS_MULTIPLEUSE with no override. This makes an override for it by taking advantage
+// of its existing virtual declarations.

```

Comment:
Wait hold on, why exactly do we want single use?

Is 
* the inbox conhost going to look up the registered delegation console, 
* find an OpenConsole.exe, 
* start `openconsole.exe -Embedding` 
* then this openconsole will register `CConsoleHandoff` once
* conhost will then CoCreate the IConsoleHandoff and hand off to this openconsole
* this openconsole's `SINGLEUSE` `CConsoleHandoff` will expire, so the next time COM needs a `IConsoleHandoff`, it'll need to start a new one?
* this openconsole will do something with the connection (like hand off to a terminal)

okay I think I follow now that I wrote it up.

Reply:
OK I added a comment like this down near line 210 in the next revision where I use this template instead of where I declare it up here.